api/openapi-spec/swagger.json

```diff
@@ -1925,7 +1925,14 @@
     }
    },
    "v1.DiskTarget": {
+    "required": [
+     "bus"
```

Comment:
This shouldn't be a required argument.

Reply:
We can infer the "bus" value from the VirtualMachine Spec, nut we need one set of attributes to distinguish what the VM is gonna used for.
The simplest example is the guest OS. Assuming we use a Q35 chipset (as we should):
- windows guest OS: -> bus = sata
- Linux (any) guest OS: -> bus = virtio

So the question becomes: what we can reliably use as set of attributes?

Reply:
For as long as we don't have the presets in place, let's default to the more compatible buses. That means imo sata if 135 if selected and probably ide if it is an older machine type.

Reply:
Done